Description: add shell script testing.sh for echoing a test message
 use bzr add testing.sh
Author: <name and email of author, optional>
Origin: <upstream|backport|vendor|other>, <URL, required except if Author is present>
Bug: <URL to the upstream bug report if any, implies patch has been forwarded, optional>
Bug-<Vendor>: <URL to the vendor bug report if any, optional>
Forwarded: <URL|no|not-needed, useless if you have a Bug field, optional>
Applied-Upstream: <version|URL|commit, identifies patches merged upstream, optional>
Reviewed-by: <name and email of a reviewer, optional>
Last-Update: <YYYY-MM-DD, last update of the meta-information, optional>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff -Naur hello-2.10/src/testing.sh hello-2.10.modify/src/testing.sh
--- hello-2.10/src/testing.sh	1970-01-01 08:00:00.000000000 +0800
+++ hello-2.10.modify/src/testing.sh	2023-01-16 10:30:42.234287160 +0800
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -e
+echo "this is a test from Hua Guo"
+exit 0
